documentation/user_docs/docs/Contributing.rst

```diff
@@ -0,0 +1,41 @@
+======================
+Contributing to Horace
+======================
+
+Contributing documentation
+==========================
+
+If you wish to contribute directly to the documentation, please follow the 5 step process below:
+
+1. If you notice a mistake or a place which could be improved, click the ``Click here to edit`` button at the bottom of
+   the page. This will take you to the GitHub edit page (requires `GitHub account <https://github.com/login>`__).
+2. Correct the mistake in the GitHub editor (see `reST documentation
+   <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`__). **N.B.** it is possible to click the
+   ``Preview`` tab at the top to see an approximate render of your changes.
+3. Select the ``Create a new branch for this commit`` option at the bottom of the page, with comments describing your
+   changes as appropriate in the "Add an optional extended description" box above and click the ``Propose changes`` button.
+4. In the ``labels`` tab on the right-hand panel select ``documentation`` and ``DO_NOT_CI`` labels. This makes it
+   possible for the development team to track the changes more easily.
+5. Click ``Create pull request``. This will submit the changes to the development team for review.
```

Comment:
I really don't think that we want to confuse non-expert users (which this is directed at) with anything to do with that process, that is internal and already established within the team. As the PR owner, the user will be informed when it is merged automatically unless they have explicitly disabled emails. 

As for promising a timescale, I'm not sure we guarantee that, in general this will be one file at a time, and very quick to review (i.e. no functionality), but we don't make promises on turnaround elsewhere.